content/en/learn.md

```diff
@@ -3,108 +3,85 @@ title: Learn
 sidebar: false
 ---
 
+***Welcome to learning about NumPy!***
 
-**Side Navigation Bar:**
+To learn how to use NumPy, you could begin with the resources listed under the 'beginners' section. Once you have a broad view into how to get started with using NumPy, you could try the more involved tutorials and resources listed under 'advanced' section.
 
-[Citing NumPy](github.com/numpy/numpy.org/blob/master/content/en/citing-numpy.md)
+This is a curated collection of NumPy related educational resources. Some are very specific to NumPy, while others offer a broader view on numerical computing. It is a continuously evolving list. In case you would like to contribute, please refer to the section 'Contributing to NumPy Learning Resources' below. 
+***
 
-[Get Help](github.com/numpy/numpy.org/blob/master/content/en/numpy-help.md)
+## Beginners
 
-**__________________**
+There is tons of information about NumPy out there. If you are new to NumPy, we'd strongly recommend the following tutorials, books and videos to get started.
 
+<i class="fad fa-chalkboard"></i> **Tutorials**
```

Comment:
That's strange.  The Hero theme uses font awesome (fa) so these classes/icons can be referenced from within hugo subsystem (I think).  I did not do any additional css download/update for these.  Let me check what else could be the problem.  There are other files in website content which refer to fontawesome - see list below:

./layouts/partials/javascript.html
./.git/modules/themes/hugo-fresh/index
./themes/hugo-fresh/resources/_gen/assets/sass/style.sass_cf66e63debe6917c04534d6c7b66f623.content
./themes/hugo-fresh/static/css/icons.css

Reply:
This is how it shows up on my Mac - using default font awesome available with Hugo hero theme.

<img width="1030" alt="Screenshot 2019-12-30 at 7 17 18 AM" src="https://user-images.githubusercontent.com/5890484/71565416-9fc2d280-2ad4-11ea-83b2-1fdeef97dd65.png">
<img width="945" alt="Screenshot 2019-12-30 at 7 17 32 AM" src="https://user-images.githubusercontent.com/5890484/71565420-aea98500-2ad4-11ea-823a-9247010b66ef.png">
<img width="903" alt="Screenshot 2019-12-30 at 7 17 44 AM" src="https://user-images.githubusercontent.com/5890484/71565423-b5d09300-2ad4-11ea-9270-2cd1e6745192.png">

Reply:
Hmm very odd. On my Macbook it looks fine as well. Don't have access to the Linux machine where I saw the issue for a couple of days. Since everything else looks good, let's merge it now and I'll investigate later.